app/views/tasks/index.html.haml

```diff
@@ -1,5 +1,8 @@
 %h1 Tasks
-
+%h2
+  =link_to "Completed", tasks_index_path(completed: "completed")
+  =link_to "Incompleted", tasks_index_path(completed: "incompleted")
```

Comment:
If there aren't any incomplete or completed tasks will u still want to display these links?

Reply:
Good hint. Added an <code>if</code> condition.